android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/Addon.kt

```diff
@@ -280,35 +291,73 @@ data class Addon(
          * @param installedState optional - an installed state.
          */
         fun newFromWebExtension(extension: WebExtension, installedState: InstalledState? = null): Addon {
-            val name = extension.getMetadata()?.name ?: extension.id
-            val description = extension.getMetadata()?.description ?: extension.id
-            val permissions = extension.getMetadata()?.permissions.orEmpty() +
-                extension.getMetadata()?.hostPermissions.orEmpty()
-
-            val developerName = extension.getMetadata()?.developerName.orEmpty()
+            val metadata = extension.getMetadata()
+            val name = metadata?.name ?: extension.id
+            val description = metadata?.description ?: extension.id
+            val permissions = metadata?.permissions.orEmpty() +
+                metadata?.hostPermissions.orEmpty()
+
```

Comment:
nit: this blank line is probably unnecessary.

Reply:
Removed!